_posts/2022-07-28-codeception-5.markdown

```diff
@@ -70,9 +70,11 @@ extensions:
 Next steps:
 
 1. Rename your suite configuration files:
+    ```
     `acceptance.suite.yml` => `Acceptance.suite.yml`
     `functional.suite.yml` => `Functional.suite.yml`
-    `unit.suite.yml` => `Unit.suite.yml`
+    `unit.suite.yml`       => `Unit.suite.yml`
+    ```
```

Comment:
I think that it would be better to remove backticks from file names, because they aren't used for formatting inside code block.